src/test/java/AddSongToPlaylist.java

```diff
@@ -0,0 +1,58 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class AddSongToPlaylist extends BaseTest {
+
+    @Test
+    public static <Actions> void AddSongToPlaylistTest() throws InterruptedException {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io/";
+        String urlHome = "https://bbb.testpro.io/#!/home";
+
+        driver.get(url);
+        //These are the elements that we need to access the CSS selector:
+
+        WebElement loginField = driver.findElement(By.cssSelector("[type='email']"));
+        WebElement passwordField = driver.findElement(By.cssSelector("[placeholder='Password']"));
+        WebElement submitLogin = driver.findElement(By.cssSelector("[type='submit']"));
+
+        //WHEN user types in their email address
+
+        loginField.sendKeys("janezelenova@gmail.com");
+        passwordField.click();
+        passwordField.sendKeys("Floridaliving2023$");
+        submitLogin.click();
+
+        Thread.sleep(5000);
+
+        driver.get(urlHome);
+        driver.manage().timeouts().implicitlyWait(10000, TimeUnit.MILLISECONDS);
+
+
+        WebElement from = driver.findElement(By.cssSelector("[class=details]"));
+        WebElement to = driver.findElement(By.cssSelector("[href=#!/playlist/33391]"));
+
+        //now drag and drop
+
+        Actions builder = new Actions(driver);
+
+        builder.dragAndDrop(from, to).perform();
+
+        Thread.sleep(5000);
+
+
+        Assert.assertEquals(driver.getCurrentUrl(), urlHome);
+        Assert.assertNotEquals(driver.getCurrentUrl(), url);
```

Comment:
Test failed! You got this problem, please correct it. 
![image](https://user-images.githubusercontent.com/104761482/221787732-a20edae2-c312-4572-a714-e72e06f45d98.png)

Hint

![image](https://user-images.githubusercontent.com/104761482/221787851-c03d4e02-3d83-48b2-ac1e-09ec6289d20f.png)

![image](https://user-images.githubusercontent.com/104761482/221787988-4fb8f041-2654-4421-b8d0-14c6fb87b9f6.png)